apps/web/components/Cards/RegisterCard/RegisterCard.tsx

```
@@ -0,0 +1,145 @@
+        <CardContent>
+          <div className="flex flex-col gap-4">
+            <div className="flex flex-col gap-2">
+              <Label htmlFor="name">{t("common.username")}</Label>
+              <Input
+                id="name"
+                type="text"
+                placeholder={t("common.username")}
+                {...register("name")}
+              />
+              {errors.name && (
+                <p className="text-sm text-red-500">{t(errors.name.message as any)}</p>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)